lib/nationbuilder/method.rb

```diff
@@ -40,7 +40,13 @@ def method_args(args)
   def nonmethod_args(args)
     a = {}
```

Comment:
So I realize you didn't write this part, but it might be a good time for a small refactoring. If you ever have the pattern

```
foo = ..
something.each do
   play with foo
end
foo
``` 
you can _probably_ replace it with `each_with_object`, e.g.

```Ruby
def nonmethod_args(args)
  args.each_with_object({}) do |(k, v), memo|
    if k == :custom_values
      v.each do |custom_k, custom_v|
        memo[:"custom_values[#{custom_k.to_s}]"] = custom_v
      end
    else
      memo[k] = v unless parameters.include?(k)
    end
  end
end
```

Reply:
Looking more into this - the issue seems to be the query params not being encoded properly (ie. the nested hash encoding as a hash, not as the server expects). So what's needed is effectively something along the lines of `ActiveSupport` [`to_query`](https://apidock.com/rails/Hash/to_query) or `Rack`'s [`build_nested_query`](http://www.rubydoc.info/gems/rack/Rack/Utils#build_nested_query-class_method). 

Without adding additional dependencies, this could be done in Ruby following the design in `Rack` or akin to [how it's done here](https://mensfeld.pl/2012/01/converting-nested-hash-into-http-url-params-hash-version-in-ruby/) (perhaps as a `Utils` module), though it feels like that should be applied within `client` when [we append the args to the query](https://github.com/nationbuilder/nationbuilder-rb/blob/master/lib/nationbuilder/client.rb#L57) instead of handling it as a special case in `Method` - thoughts?